quickstarts/kubernetes-opentelemetry/config.yml

```
@@ -0,0 +1,54 @@
+slug: kubernetes-opentelemetry
+title: Kubernetes (OpenTelemetry)
+description: |
+  ## Why monitor Kubernetes?
+
+  Kubernetes is an open-source system for automating deployment, scaling, and, management of containerized applications. The New Relic Kubernetes monitoring quickstart gives you visibility into your Kubernetes clusters and workloads in minutes, whether your clusters are hosted on-premises or in the cloud.
+
+  ### Kubernetes quickstart highlights
+
+  The New Relic Kubernetes quickstart uses dashboards to proactively monitor your metrics, like:
+
+  - resources used
+  - number of K8s objects
+  - namespaces per cluster
+  - pods by namespace
+  - container cpu usage
+  - container restarts
+  - missing pods by deployment
+  - node resource consumption, and more.
+
+  This quickstart is also compatible with on-host integrations like:
+
+  - Cassandra
+  - MySQL
+  - Apache, and more.
+
+  ### New Relic + Kubernetes = Optimum performance monitoring
+
+  The [New Relic Kubernetes](https://docs.newrelic.com/docs/integrations/kubernetes-integration/installation/kubernetes-integration-install-configure/) quickstart has multiple components that work together to give you end-to-end observability across your clusters. While you have the flexibility to deploy the components that you prefer, to achieve full observability, you need to install the complete package to monitor all metrics. Use our quickstart to generate a Kubernetes manifest and add Pixie for more fine-grained telemetry data. You can also do the installation with Pixie for fine-grained telemetry data.
+  Our quickstart monitors the aggregated core and memory usage across all nodes in your cluster. This allows you to meet resource requirements for optimal application performance. It also empowers you to track resource consumption, find pods that aren't running, monitor disk usage, and troubleshoot container restarts. The New Relic Kubernetes integration has dashboards and a cluster explorer that provide a multi-dimensional representation of a Kubernetes cluster from which you can explore your namespaces, deployments, nodes, pods, containers, and applications. Download the New Relic Kubernetes quickstart today to gain instant visibility into your Kubernetes services, clusters and workloads in minutes.
+summary: |
+  Monitoring Kubernetes is crucial to gain instant visibility into Kubernetes clusters and workloads. Download New Relic Kubernetes quickstart to proactively monitor Kubernetes cluster health and capacity.
+level: New Relic
+authors:
+  - New Relic
+documentation:
+  - name: Kubernetes installation docs
+    description: |
+      Kubernetes is an open-source container-orchestration system for automating
+      computer application deployment, scaling, and management.
+    url: >-
+      https://docs.newrelic.com/docs/integrations/host-integrations/host-integrations-list/kubernetes-monitoring-integration
+dataSourceIds:
+  - kubernetes
```

```suggestion
  - kubernetes-opentelemetry
```
Since this quickstart is different from Kubernetes, you need to create a new datasource: `kubernetes-opentelemetry`. Here is the example: https://github.com/newrelic/newrelic-quickstarts/tree/release/data-sources/apigee-api

---

Hi @sjyothi54. Do the `dataSourceId` and `dashboards` properties require a value or can I remove them from the config?

---

Hey @douavue, dashboards are optional, but it is best practice to provide them in order to show the customer what are the metrics they get in dashboard. Since this is a new quickstart, datasourceIds is required. Please provide datasourceIds for the Kubernets opentelemetry, it is used to create install steps in production.

---

@sjyothi54 we don't provide a dashboard, because the K8s OTel instrumentation enables all the regular K8s UIs so no additional dashboards are needed.

---

> @sjyothi54 we don't provide a dashboard, because the K8s OTel instrumentation enables all the regular K8s UIs so no additional dashboards are needed.

@mangulonr @douavue Thanks for the clarification